python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -967,10 +970,16 @@ def _setFlushed(self):
         """Un-Set the dirty flag all of the way down the schema."""
         if self._schema:
             for k in self._schema.allItemNames():
+                ## Avoid attributes the likes of job.master which crawl back up the tree
+                if self._schema[k].hasProperty('visitable'):
+                    if (not self._schema[k].getProperties()['visitable'] or self._schema[k].getProperties()['visitable'] == 0):
+                        continue
+                if self._schema[k].hasProperty('transient'):
+                    if(self._schema[k].getProperties()['transient'] or self._schema[k].getProperties()['transient'] == 1):
```

Comment:
I hold particular bias against not using ```is True``` and ```is not True```, on a few projects I've seen much more time lost to problems introduced by things being equivalent to ```True``` than time saved by duck-type programming and design issues but this is delving into the realms of personal preference here quite strongly I think